src/installer.ts

```diff
@@ -97,23 +97,23 @@ export function downloadDub(env) {
 	var url = "";
 	var ext = "";
 	if (process.platform == "linux" && process.arch == "x64") {
-		url = "https://code.dlang.org/files/dub-1.0.0-linux-x86_64.tar.gz";
+		url = "https://code.dlang.org/files/dub-1.8.0-linux-x86_64.tar.gz";
 		ext = ".tar.gz";
 	}
 	else if (process.platform == "linux" && process.arch == "ia32") {
-		url = "https://code.dlang.org/files/dub-1.0.0-linux-x86.tar.gz";
+		url = "https://code.dlang.org/files/dub-1.8.0-linux-x86.tar.gz";
 		ext = ".tar.gz";
 	}
 	else if (process.platform == "linux" && process.arch == "arm") {
-		url = "https://code.dlang.org/files/dub-1.0.0-linux-arm.tar.gz";
+		url = "http://code.dlang.org/files/dub-1.1.0-linux-arm.tar.gz";
```

Comment:
oh ok I sent http but it is also available under https, can you make this https again?